packages/spacecat-shared-http-utils/test/auth/auth-wrapper.test.js

```diff
@@ -54,16 +55,20 @@ describe('auth wrapper', () => {
       },
       dataAccess: {},
     };
-    mockApiKey = createApiKey({
-      hashedKey: '372c6ba5a67b01a8d6c45e5ade6b41db9586ca06c77f0ef7795dfe895111fd0b',
-      name: 'Test API key name',
-      scopes: [
+    mockApiKey = {
+      getId: () => '1C4ED8DE-8ECD-42E1-9812-AF34082FB1B4',
+      getHashedKey: () => '372c6ba5a67b01a8d6c45e5ade6b41db9586ca06c77f0ef7795dfe895111fd0b',
```

Comment:
Just confirming: These mocked functions (`getHashedKey` in particular) will be removed in the next PR?

Reply:
Yes - correct. The next set of modifications will be for `spacecat-shared-utils`, so that it uses the refactored getHashedApiKey from `spacecat-shared-data-access`.